lib/v2/luogu/daily.js

```diff
@@ -1,12 +1,6 @@
 const got = require('@/utils/got');
 const cheerio = require('cheerio');
 const { parseDate } = require('@/utils/parse-date');
-const md = require('markdown-it')({
```

Comment:
Remove markdown parse, it takes no effects to final result.
It just use for get a link to actual link.